src/DocumentFormat.OpenXml/OpenXmlDomReader.cs

```diff
@@ -89,7 +89,9 @@ public override ReadOnlyCollection<OpenXmlAttribute> Attributes
                 {
                     if (_emptyReadOnlyList == null)
                     {
-                        _emptyReadOnlyList = new ReadOnlyCollection<OpenXmlAttribute>(new List<OpenXmlAttribute>());
+#pragma warning disable CA1825 // Because we also need to support the .NET Framework versions below 4.6, and these versions do not exist Array.Empty method
```

Comment:
There is a utility for that in the SDK: `Cached.Array<OpenXmlAttribute>()`.

Reply:
@twsouthwick Thank you and I fixed it